docs/product/components/navigation.html

```diff
@@ -223,6 +224,150 @@
     </div>
 </div>
 
+<section class="stacks-section">
+    {% header "h2", "Interactive tabs" %}
+
+    <p class="stacks-copy">Stacks provides a Stimulus controller for interactive tab-style navigation within a single document.</p>
+
+    {% header "h3", "Attributes" %}
+    <div class="overflow-x-auto mb32">
```

Comment:
Docs look good overall. The table of "add all these things" looks a little long to me, though. I think we could easily transition to a paragraph saying "make sure you add all these aria attributes and make your markup look like this", leaving the controller values in the table.

nit: I don't see a section for the fired events, like we have for [popovers](https://stackoverflow.design/product/components/popovers/#interactive-events).

Reply:
Done